app_dart/lib/src/foundation/utils.dart

```diff
@@ -88,7 +89,28 @@ Future<RepositorySlug> repoNameForBuilder(List<Map<String, dynamic>> builders, S
 Future<List<Map<String, dynamic>>> getBuilders(HttpClientProvider branchHttpClientProvider, Logging log,
     GitHubBackoffCalculator gitHubBackoffCalculator, String bucket) async {
   final String filename = bucket == 'try' ? 'luci_try_builders.json' : 'luci_prod_builders.json';
-  String builderContent = await remoteFileContent(branchHttpClientProvider, log, gitHubBackoffCalculator, filename);
+  String builderContent = await remoteFileContent(
+      branchHttpClientProvider, log, gitHubBackoffCalculator, '/flutter/cocoon/master/app_dart/dev/$filename');
+  builderContent ??= '{"builders":[]}';
+  Map<String, dynamic> builderMap;
+  try {
+    builderMap = json.decode(builderContent) as Map<String, dynamic>;
+  } on FormatException catch (e) {
+    log.error('error: $e');
+    builderMap = <String, dynamic>{'builders': <dynamic>[]};
+  }
+  final List<dynamic> builderList = builderMap['builders'] as List<dynamic>;
+  return builderList.map((dynamic builder) => builder as Map<String, dynamic>).toList();
+}
+
+/// Gets supported luci builders based on [bucket] and [repo] via GitHub http request.
+Future<List<Map<String, dynamic>>> getRepoBuilders(HttpClientProvider branchHttpClientProvider, Logging log,
+    GitHubBackoffCalculator gitHubBackoffCalculator, String bucket, String repo) async {
+  String filePath = repo == 'engine' ? '$repo/master/ci/dev/' : '$repo/master/dev/';
+  final String fileName = bucket == 'try' ? 'try_builders.json' : 'prod_builders.json';
```

Comment:
Do we use prod builders for something? I think those are using LUCI CQ

Reply:
Yes. https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/service/luci.dart#L198

Reply:
Are we using that function somewhere? it may be that we don't need the prod files

Reply:
Yes, we need it to update sdk luci status on dashboard.